### DISCUSSION TITLE
How to Find the Inspection ID

### DISCUSSION THREAD

**Discussion by sundaycrafts** [JetBrains/Qodana#230] [Q&A]
Hello there,

Does anyone know how can I find the inspection id? I couldn't figure it out from the official documentation, perhaps because I lack some prerequisite knowledge that Qodana assumes. Could someone please advise me?

**Comment by sundaycrafts** [comment]
bump

**Comment by tiulpin** [comment]
- You can enable/disable inspections in Qodana UI – this way, the inspection IDs will be present in the suggested `qodana.yaml` configuration
- You can save the inspection profile in your IDE and find the profile in `.idea` directory, where inspection IDs would be written
- You can explore https://www.jetbrains.com/pages/inspectopedia-by-jetbrains/
